src/html.sortable.js

```diff
@@ -290,6 +290,7 @@ var _enableSortable = function (sortableElement) {
   var items = _filter(_getChildren(sortableElement), opts.items)
   var handles = _getHandles(items, opts.handle)
   _attr(sortableElement, 'aria-dropeffect', 'move')
+  _data(sortableElement, '_disabled', 'false')
```

Comment:
Ahaa, I totally understand that, types are not fun, especially if you come from a strongly typed language. 😁 

Anyway, our `data` has nothing to do with the html data element. It is stored on `$sortable.h5s.data` so types are supported.